ruskeyverbs/templates/ruskeyverbs/answer_evaluation.html

```diff
@@ -11,20 +14,43 @@ <h2 class="incorrect">Incorrect</h2>
 <p>The correct answer is: <span class="bold">{{ answer }}</span></p>
 <p class="center">{{ russian_text }}</p>
 <h3>Score: {{ score }}</h3>
+{% if score <= 90 %}
+  <p>Retype the correct answer (with punctuation and capitalization):</p>
+  <input type="text" name="" v-model="userinbox" value="" v-on:keyup='CompareAns'></input>
+{% endif %}
 {% if quiz_state == 1 %}
```

Comment:
This whole conditional block is pretty ugly. It was ugly before this revision and I think got uglier still here. The intent is to show a single Continue button and have the url and disabled values for it set conditionally based on the state of the quiz, right? One better strategy might be to calculate the proper URL and disabled state in the view method and pass those two values to the template so that the template could just have one definition of the button like:

<a id="next" href="{% next_url %"} {% if next_disabled %}disabled{% endif %})Continue</a>